test/extended/util/docker.go

```diff
@@ -0,0 +1,60 @@
+package util
+
+import (
+	dockerClient "github.com/fsouza/go-dockerclient"
+	tutil "github.com/openshift/origin/test/util"
+)
+
+func CorruptImage(corruptee, corruptor string) error {
+	client, dcerr := tutil.NewDockerClient()
+	if dcerr != nil {
+		return dcerr
+	}
+	opts := dockerClient.TagImageOptions{
+		Repo:  corruptee,
+		Tag:   "latest",
+		Force: true,
+	}
+	terr := client.TagImage(corruptor, opts)
+	return terr
+}
+
+func ResetImage(name string) error {
+	client, dcerr := tutil.NewDockerClient()
+	if dcerr != nil {
+		return dcerr
+	}
+	opts := dockerClient.PullImageOptions{
+		Repository: name,
+		Tag:        "latest",
+	}
+	perr := client.PullImage(opts, dockerClient.AuthConfiguration{})
+	return perr
+}
+
+func CompareImages(comp1, comp2 string, returnID1, returnID2 *string) error {
```

Comment:
Yep, aware of the multiple return values.  I still don't see yet the huge advantage for that in this case, but perhaps my "conversion" to Go is not complete ;-) I'll ponder, but am not sure yet about swtiching.  And yes, I was just trying to avoid a call - but upon reflection agree being more generic (even if it is 2 calls in the short term) is more flexible.  I'll switch that and the name minimally, and ponder the return type.  Come back at me again on that one depending on how strongly you feel.